### PR TITLE
Initial adjustments post rebase

### PR DESCRIPTION
**Pull request by Timo614**
Some minor adjustments / new methods for https://github.com/big-neon/bn-api/pull/2.

Putting up a PR now as it's referenced by that branch but still wish to flesh this out a bit more with additional methods, etc. prior to merging.

### CODE REVIEW

File: src/models/artists.rs [hunk 1]
```diff
@@ -36,4 +36,33 @@ impl Artist {
             name: String::from(name),
         }
     }
+
+    pub fn all(conn: &Connectable) -> Vec<Artist> {
+        artists::table
+            .load(conn.get_connection())
+            .expect("Unable to load artists")
```
Comment (mikethetike) [inline src/models/artists.rs]: Can we use `DatabaseError::wrap()` here instead of `expect`

**Comment by Timo614** [comment]
I've adjusted this as per the review feedback to use the DatabaseError structs now for the db responses. I've also added a new API branch to go along with this one as the interface has changed now: https://github.com/big-neon/bn-api/pull/3

File: tests/unit/artists.rs [hunk 2]
```diff
@@ -1,11 +1,68 @@
...
+    assert_eq!(name, artist.name);
+    assert_eq!(artist.id.to_string().is_empty(), false);
+
+    let found_artist = Artist::find(&artist.id, &project).unwrap();
+    assert_eq!(found_artist.id, artist.id);
+    assert_eq!(found_artist.name, artist.name);
+}
+
+#[test]
+fn all() {
```
Comment (CjS77) [inline tests/unit/artists.rs]: Should add a test that checks that `all` works for n>1 artists too

Comment (Timo614) [reply tests/unit/artists.rs]: Sounds good, added it to the test.

**Pull request merged**